src/crawlee/playwright_crawler/_playwright_crawler.py

```diff
@@ -168,9 +172,7 @@ async def _navigate(
         async with context.page:
             if context.request.headers:
                 await context.page.set_extra_http_headers(context.request.headers.model_dump())
-            # Navigate to the URL and get response.
-            response = await context.page.goto(context.request.url)
-
+            response = await context.page.goto(context.request.url, wait_until='domcontentloaded')
```

Comment:
By default, Playwright waits until the `load` event (i.e. until the HTML has loaded and all the subresources - images, stylesheets - loaded too). 

I don't have a strong opinion here; I'm just curious - did this make a change in some behavior during the development? 👀

Reply:
No, I forgot it there when I was testing stuff. It is not needed. Good catch.